docs/source/usrman/env_var.rst

```diff
@@ -0,0 +1,52 @@
+.. _environment:
+
+Environment variables
+=====================
+
+Below we list the environment variables added since mpi4py v3.1.0. The environment
+variables overwrite their corresponding Python attributes at the ``MPI`` import time.
+
+``MPI4PY_RC_INITIALIZE``
+  Default: ``True``
+
+  Whether or not to automatically initialize MPI at the ``MPI`` import time. Corresponding
+  to ``mpi4py.rc.initialize``.
+
+``MPI4PY_RC_FINALIZE``
+  Default: ``None``
+
+  Whether or not to automatically finalize MPI at the exit time. Corresponding to
+  ``mpi4py.rc.finalize``.
+
+``MPI4PY_RC_THREADS``
+  Default: ``True``
+
+  Whether or not to initialize MPI with thread support. Corresponding to ``mpi4py.rc.threads``.
+  
+``MPI4PY_RC_THREAD_LEVEL``
+  Default: ``'multiple'``
+    
+  The level of required thread support. Any of the following is accepted: ``"single"``,
+  ``"funneled"``, ``"serialized"``, ``"multiple"``. Corresponding to ``mpi4py.rc.thread_level``.
+
+``MPI4PY_RC_FAST_REDUCE``
+  Default: ``True``
+
+  Whether or not to use tree-based reductions for objects. Corresponding to ``mpi4py.rc.fast_reduce``.
+
+``MPI4PY_RC_RECV_MPROBE``
+  Default: ``True``
+
+  Whether or not to use matched probes to receive objects. Corresponding to ``mpi4py.rc.recv_mprobe``.
+
+``MPI4PY_RC_ERRORS``
+  Default: ``'exception'``
+
+  Controls mpi4py's error handling policy. Any of the following is accepted: ``'exception'``,
+  ``'default'``, ``'fatal'``. Corresponding to ``mpi4py.rc.errors``.
+
+``MPI4PY_PICKLE_PROTOCOL``
+  Default: ``5``.
+
+  Controls the default pickle protocol to use when communicating Python objects. Corresponding
+  to ``MPI.pickle.PROTOCOL``.
```

Comment:
*Corresponds to the `mpi4py.pickle.PROTOCOL` attribute.*

Reply:
Hmmm I checked this attribute exists and found it from the doc you wrote:
https://github.com/mpi4py/mpi4py/blob/60ce05d567bda91e6c481851772ada7d921e847f/docs/source/usrman/tutorial.rst#L45-L47
Would you like me to change that part of description as well?

Reply:
Oh, this description in the tutorial is even better. It is indeed the `PROTOCOL` attribute of the `MPI.pickle` object.

Reply:
Let's leave it as it is now.